dockstore-webservice/src/main/resources/migrations.1.5.0.xml

```diff
@@ -30,4 +30,44 @@
         <addNotNullConstraint columnDataType="clob" columnName="referencetype" tableName="tag"/>
         <addNotNullConstraint columnDataType="clob" columnName="referencetype" tableName="workflowversion"/>
     </changeSet>
+
+    <!-- adding support for input and output format -->
+    <changeSet author="gluu (generated)" id="1525277022753-5">
+        <createTable tableName="fileformat">
+            <column autoIncrement="true" name="id" type="BIGSERIAL">
+                <constraints primaryKey="true" primaryKeyName="fileformat_pkey"/>
+            </column>
+            <column name="value" type="VARCHAR(255)"/>
+        </createTable>
+    </changeSet>
+    <changeSet author="gluu (generated)" id="1525277022753-6">
+        <createTable tableName="version_input_file_format">
+            <column name="versionid" type="BIGINT">
+                <constraints primaryKey="true" primaryKeyName="version_input_file_format_pkey"/>
+            </column>
+            <column name="fileformatid" type="BIGINT">
+                <constraints primaryKey="true" primaryKeyName="version_input_file_format_pkey"/>
+            </column>
+        </createTable>
+    </changeSet>
+    <changeSet author="gluu (generated)" id="1525277022753-7">
+        <createTable tableName="version_output_file_format">
+            <column name="versionid" type="BIGINT">
+                <constraints primaryKey="true" primaryKeyName="version_output_file_format_pkey"/>
+            </column>
+            <column name="fileformatid" type="BIGINT">
+                <constraints primaryKey="true" primaryKeyName="version_output_file_format_pkey"/>
+            </column>
+        </createTable>
+    </changeSet>
+    <changeSet author="gluu (generated)" id="1525277022753-8">
+        <addUniqueConstraint columnNames="value" constraintName="uk_eqofp3o961e4nqeehch1yhai" tableName="fileformat"/>
```

Comment:
Ah good, there's a constraint. But do name it something more descriptive.

Reply:
just follow the example from 1.3.0.generated :)

Reply:
A generated migration obviously can't come up with nice names, but we as humans can.

Reply:
Do you want me to change the ones in 1.3.0.generated too?

Reply:
It's worth an issue, but I don't think you need to get sidetracked now.